libcontainer/cgroups/fs/fs.go

```diff
@@ -241,6 +241,15 @@ func (m *manager) Path(subsys string) string {
 	return m.paths[subsys]
 }
 
+func (m *manager) Kill() error {
+	m.mu.Lock()
+	defer m.mu.Unlock()
+	// Since its not a subsystem and we dont want to regenerate path using pid
```

Comment:
nits:
* `its` -> `it's`
* `dont' -> `don't`

Reply:
Fixed